src/schema/v2/me/newWorksFromGalleriesYouFollow.ts

```diff
@@ -45,3 +45,12 @@ export const newWorksFromGalleriesYouFollow: GraphQLFieldConfig<
     }
   },
 }
+
+// Because the loader doesn't return the total size, we use this way to calculate it for the connection.
```

Comment:
Feel free to make a change upstream in Gravity to make any upstream endpoints used more 'properly' perform pagination + return counts.

Reply:
Later Friday afternoon fix to get any result from the connection 😮‍💨 

Thanks for pushing for a properly working solution. I just had to pass `total_count: true` as a param to the followedProfilesArtworksLoader to make it work (https://github.com/artsy/metaphysics/pull/4854/commits/fd97f6cc6d813f2af8cb1e480151e567fe11f77d).